db/seeds/dev/palettepicker.js

```diff
@@ -0,0 +1,34 @@
+
+exports.seed = async (knex) => {
+  try {
+    await knex('palettes').del()
+    await knex('projects').del()
+
+    const projectId = await knex('projects').insert({
+      name: 'Project One'
+    }, 'id')
+    return knex('palettes').insert([
+      { 
+        name: 'Palette One', 
+        color_one: '#47850',
+        color_two: '#47851',
+        color_three: '#47852',
+        color_four: '#47853',
+        color_five: '#47854',
+        project_id: projectId[0]
+      },
+      { 
+        name: 'Palette Two', 
+        color_one: '#47850',
+        color_two: '#47851',
+        color_three: '#47852',
+        color_four: '#47853',
+        color_five: '#47854',
+        project_id: projectId[0]
+      }
+    ])
+  } catch (error) {
```

Comment:
Nice work seeding some fake data to work with while we work on the endpoints - especially since we started with GET endpoints